### DISCUSSION TITLE
blog/blog-refurbishment/

### DISCUSSION THREAD

**Discussion by giscus** [152334H/152334H.github.io#3] [Announcements]
# blog/blog-refurbishment/

Under new management

https://152334h.github.io/blog/blog-refurbishment/

**Comment by 152334H** [comment]
test

**Comment by wemoveon2** [comment]
Whoa, someone in the military writing about SOTA ML papers and using Rust? Are you in the space force?

**Reply by 152334H** [reply]
🤷 not like I do most of those things anymore

**Comment by KatsuragiCSL** [comment]
Really nice, I thought it is papermod + https://utteranc.es/ at the first glance. Your setup looks even nicer!

**Reply by 152334H** [reply]
it's buggy in a lot of places and I wouldn't advise adopting it. If I could pause time I would switch to some obsidian publish equivalent.
